frontend/public/components/monitoring/metrics.tsx

```diff
@@ -310,6 +310,22 @@ export const QueryTable: React.FC<QueryTableProps> = ({ index, namespace }) => {
     observe.getIn(['queryBrowser', 'lastRequestTime']),
   );
 
+  const isDisabledSeriesEmpty = useSelector(({ observe }: RootState) =>
+    _.isEmpty(observe.getIn(['queryBrowser', 'queries', index, 'disabledSeries'])),
+  );
+
+  const dispatch = useDispatch();
+
+  const toggleAllSeries = React.useCallback(
```

Comment:
I think it would be worth converting this into a Redux action & reducer in `frontend/public/actions/observe.ts` and `frontend/public/reducers/observe.ts`. Then both this code and the duplicate code in `QueryKebab` can use that single action.

We can get `isDisabledSeriesEmpty` and `series` inside the action, so we only need to pass in `index`.

Reply:
👍 Updated in the latest commit